src/traits/PrivacyTrait.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace luya\privacy\traits;
+
+use Yii;
+use yii\web\Cookie;
+
+/**
+ * Trait PrivacyTrait
+ * 
+ * The privacy trait helps getting and setting the privacy policy state.
+ * 
+ * @author Alex Schmid <alex.schmid@stud.unibas.ch>
+ * @since 1.0.0
+ */
+trait PrivacyTrait
+{
+    /**
+     * @return bool|null The privacy value
+     */
+    public function getPrivacy()
```

Comment:
I would suggest to use another function name, which really says what it does:

getPrivacyCookieValue